include/revng/Pipes/LinkSupport.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Pipeline/ContainerSet.h"
+#include "revng/Pipeline/Contract.h"
+#include "revng/Pipeline/LLVMContainer.h"
+#include "revng/Pipeline/LLVMGlobalKindBase.h"
+#include "revng/Pipeline/Target.h"
+#include "revng/Pipes/RevngPipes.h"
+
+namespace pipeline {
+
+class LinkSupportPipe {
+public:
+  static constexpr auto Name = "LLVM Link Support Pipe";
+
+  std::array<ContractGroup, 1> getContract() const {
+    return { ContractGroup(Root,
+                           Exactness::DerivedFrom,
+                           0,
+                           InputPreservation::Preserve) };
```

Comment:
Is this actually necessary? Can't this be a `PureLLVMPipe`?

Reply:
same issue as other one above.

Reply:
ah this one cannot because it's not a pass.it invokes llvm::linkModules